BUILD_DOCS.md

```
@@ -0,0 +1,255 @@
+
+### 3. Build Static Site
+
+```bash
+mkdocs build
+```
+
+Output in `site/` directory.
+
+## Documentation Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)